.github/workflows/continuous-integration.yml

```diff
@@ -27,5 +27,5 @@ jobs:
     name: "PHPUnit"
     uses: "doctrine/.github/.github/workflows/continuous-integration.yml@7.1.0"
     with:
-      php-versions: '["7.1", "7.2", "7.3", "7.4", "8.0", "8.1", "8.2", "8.3"]'
+      php-versions: '["8.1", "8.2", "8.3"]'
       composer-options: '--ignore-platform-req=php+'
```

Comment:
Do we still need `--ignore-platform-req=php+` in this workflow? Not a blocker though.